package.json

```diff
@@ -38,7 +44,10 @@
   },
   "devDependencies": {
```

Comment:
are packages used for development purposes, e.g for running tests or transpiring your code.